docs/resources.rst

```diff
@@ -275,6 +275,13 @@ This |inspec resource| matches any keyword that is listed in the ``auditd.conf``
 
    its('log_format') { should eq 'raw' }
 
+Since all option names and values are case insensitive for ``auditd_conf``, we recommend to compare values with `cmp` instead of the `eq`.
+
+.. code-block:: ruby
+
+   its('log_format') { should cmp 'raw' }
+   its('max_log_file') { should cmp 6 }
+
```

Comment:
Could we establish cmp as the default matcher instead of eq for auditd_conf? Since the resource would be perfect for that!